src/controller/python/chip/storage/__init__.py

```diff
@@ -108,8 +108,9 @@ def Sync(self):
                 self._file = open(self._path, 'w')
             except Exception as ex:
                 logging.warn(
-                    f"Could not open {path} for writing configuration. Error:")
+                    f"Could not open {self._file} for writing configuration. Error:")
```

Comment:
```suggestion
                    f"Could not open {self._path} for writing configuration. Error:")
```
`self._file` is he thing we tried to assign to and failed, so it's not going to have a useful value.

Reply:
It depends what the reason of the failure is. In my case it was a permission issue....

Reply:
Oh sorry, I see now, picked the wrong variable. Agreed :+1: